application/libraries/services/NotificationService.php

```
@@ -0,0 +1,283 @@
+        }
+    }
+
+    /**
+     * 답글 알림 생성 (내 댓글에 답글)
+     *
+     * @param int $parentCommentId 부모 댓글 ID
+     * @param int $replyCommentId 답글 ID
+     * @param int $actorId 답글 작성자 ID
+     * @return void
+     */
+    public function notifyNewReply($parentCommentId, $replyCommentId, $actorId)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$replyCommentId'. (undefined)

(UnusedFormalParameter)